core/arch/arm/plat-ls/conf.mk

```diff
@@ -98,6 +98,8 @@ $(call force,CFG_PL011,y)
 $(call force,CFG_CORE_ARM64_PA_BITS,48)
 $(call force,CFG_EMBED_DT,y)
 $(call force,CFG_EMBED_DTB_SOURCE_FILE,fsl-lx2160a-qds.dts)
+$(call force,CFG_LS_I2C,y)
```

Comment:
Why do you need to force this flag? What about ```CFG_LS_I2C ?= y``` ?

Reply:
We thought of enabling it by default, but we can go for `CFG_LS_I2C ?= y`.
Will update

Reply:
> We thought of enabling it by default

That's exactly what `CFG_LS_I2C ?= y` does ;-) On the other hand, `$(call force,CFG_LS_I2C,y)` would prevent you from disabling it at build time (`make ... CFG_LS_I2C=n` would error out).

Reply:
@jforissier Yes, you are correct. My bad. :-)